### ISSUE TITLE
Bug: this app can't run on your pc

### ISSUE THREAD

**Issue by dgsmiley18** [Twig6943/Non-Bloated-Bluestacks-5#4]
I tried to open the .bat, but it gave me this error:
Specs:
Windows 11
Hyper-V enabled

Steps:
Tried to open with admin and without admin
![example](https://files.catbox.moe/sd6gi9.png)

**Comment by Twig6943** [comment]
Bro what architecture is your pc?

**Comment by Twig6943** [comment]
What architecture is your pc?

**Comment by dgsmiley18** [comment]
x86-64

**Comment by Twig6943** [comment]
I have no idea why this happens you might try another script of mine

**Comment by Plewu** [comment]
had to create account because I know whats the problem :D Mate [dgsmiley18](https://github.com/dgsmiley18) 
check if your download folder contains Bluestacks5Installer.exe (and if so check if  it has 0m)
 If so, then download lates build, put in download folder (name it as writen ealier)
 delete line 3 from bat
save as new bat, run

Twig6943 does this debloat option work only on firewall and new luncher ? so anyone can debloat same any emulator if so.

**Comment by dgsmiley18** [comment]
> had to create account because I know whats the problem :D Mate [dgsmiley18](https://github.com/dgsmiley18) check if your download folder contains Bluestacks5Installer.exe (and if so check if it has 0m) If so, then download lates build, put in download folder (name it as writen ealier) delete line 3 from bat save as new bat, run
> 
> Twig6943 does this debloat option work only on firewall and new luncher ? so anyone can debloat same any emulator if so.

This actually worked, looks like the .bat couldn't download the file and this is what caused the problem

**Comment by Plewu** [comment]
yup , im trying now to force this bat to download hyper v version, but unlucky

**Comment by secon40** [comment]
@Plewu not sure if ur still active on github but ive tried this

> had to create account because I know whats the problem :D Mate [dgsmiley18](https://github.com/dgsmiley18) check if your download folder contains Bluestacks5Installer.exe (and if so check if it has 0m) If so, then download lates build, put in download folder (name it as writen ealier) delete line 3 from bat save as new bat, run
> 
> Twig6943 does this debloat option work only on firewall and new luncher ? so anyone can debloat same any emulator if so.

not sure if youre still active on github but had the same bluestacks5installer.exe 0m problem

1. i install the zip and unzip it
2. i run the .bat as admin and your pc cant run this program thing pops up
3. bluestacks5installer.exe 0m is in downloads
4. i download a new zip, unzip it, take the .bat and edit it to delete line3, save as new, i overwrite the other install.bat, i run it and nothing happens

Not sure if im stupid and did something wrong, but ive tried many times and not sure if its a file problem or im doing something wrong

**Comment by Twig6943** [comment]
@secon40  add me on discord Twig6843
